libs/SmartStore/src/com/salesforce/androidsdk/smartstore/ui/SmartStoreInspectorActivity.java

```diff
@@ -276,7 +280,7 @@ public void onSoupsClick(View v) {
 		}
 
 		if (names.size() > 10) {
-			queryText.setText(getString(R.string.sf__inspector_soups_query));
+			queryText.setText(SOUPS_QUERY);
```

Comment:
We didn't notice until now, because this query is only used when there are more than 10 soups.